CMakeLists.txt

```diff
@@ -192,6 +205,8 @@ set(required_deps ASS
                   TagLib
                   TinyXML
                   TinyXML2
+                  try_signal
+                  Usrsctp
```

Comment:
Same as in here, only the explicit dependencies used will be in this list, a depencency of a dependency should be included as part of the top dep target (eg libtorrent)

Reply:
The problem was that a library-of-a-library wasn't linked into the final kodi.lib on Windows. Adding the dependencies here (even though they're not needed) solved the problem. `Libtorrent` and `LibDataChannel` are the only two dependencies that should be needed in CMakeLists.txt. Can you figure out how to modify the cmake for this?

Reply:
The target interfaces are wrong then (potentially from what the upstream lib exports), or you are half in/half out with target usage still.

I can, and most likely will, look into it further, but not right now.